src/pudl/transform/eiaaeo.py

```diff
@@ -676,24 +690,24 @@ class AeoCheckSpec:
 
 
 BASE_AEO_CATEGORIES = {
-    "model_case_eiaaeo": 17,
+    "model_case_eiaaeo": 20,
     "projection_year": 30,
     "electricity_market_module_region_eiaaeo": 26,
 }
 check_specs = [
     AeoCheckSpec(
         name="gen_in_electric_sector_by_tech",
         asset="core_eiaaeo__yearly_projected_generation_in_electric_sector_by_technology",
-        num_rows_by_report_year={2023: 166972},
+        num_rows_by_report_year={2023: 166972, 2025: 126750},
```

Comment:
We should probably remove all of these row count checks rather than updating them, since this is being checked in a standardized way with dbt now.